src/tools/swix/swcore/CompilerBackend/Nuget/NugetBackendCompiler.cs

```diff
@@ -53,7 +53,10 @@ public override void Generate(IEnumerable<Intermediate> intermediates, string ou
                 return;
             }
 
-            FileTransfer packageTransfer = FileTransfer.Create(null, Path.GetTempFileName(), outputPath, "NugetPackage", true);
+            int lastIndex = outputPath.LastIndexOf('.');
```

Comment:
I think the bug should be fixed in the .targets not in the tool. The tool should do what one says. The .targets should follow the "standard".